apps/dashboard/src/components/workflow-list.tsx

```diff
@@ -169,6 +169,19 @@ export const WorkflowList = () => {
                         className="cursor-pointer"
                         text={workflow.name}
                         onClick={() => {
+                          /**
+                           * Redirect to V1 legacy workflow editor if the workflow is of type REGULAR
+                           */
+                          if (workflow.type === WorkflowTypeEnum.REGULAR) {
+                            /**
+                             * React Router doesn't support nested routes, so we need to manually redirect to the legacy route
+                             */
+                            window.location.href = buildRoute(LEGACY_ROUTES.EDIT_WORKFLOW, {
+                              workflowId: workflow._id,
+                            });
+                            return;
+                          }
+
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Using `onClick` handler results in an inability to "Right-click"->"Open in new tab", it's one of my pet peeves when browsing a website. Can we use the native `href` attribute instead?‏